airbyte-workers/src/main/java/io/airbyte/workers/worker_run/TemporalWorkerRunFactory.java

```diff
@@ -72,12 +72,6 @@ public void migrateSyncIfNeeded(final Set<UUID> connectionIds) {
     final UUID connectionId = UUID.fromString(job.getScope());
     return switch (job.getConfigType()) {
       case SYNC -> () -> {
-
-        if (featureFlags.usesNewScheduler()) {
```

Comment:
why are we removing this?

Reply:
This was unused, the server directly call the temporal client with the new scheduler

Reply:
cool!